pages/hydra/developer_resources/customize_metadata/hyrax_1.0/model.md

```diff
@@ -30,14 +30,47 @@ end
 
 ## Basic and Core metadata
 
-* Basic metadata fields are defined in (app/models/concerns/hyrax/basic_metadata.rb)[https://github.com/projecthydra-labs/hyrax/blob/master/app/models/concerns/hyrax/basic_metadata.rb]
-* Core metadata fields (that should never be removed) are defined in (app/models/concerns/hyrax/core_metadata.rb)[https://github.com/projecthydra-labs/hyrax/blob/master/app/models/concerns/hyrax/core_metadata.rb]
-
-
-## Extend the model 
+### Basic metadata
+
+Basic metadata properties are defined in [app/models/concerns/hyrax/basic_metadata.rb](https://github.com/projecthydra-labs/hyrax/blob/master/app/models/concerns/hyrax/basic_metadata.rb)
+
+| Property | Predicate | Multiple |
+| -------- | --------- | -------- |
+| label | ActiveFedora::RDF::Fcrepo::Model.downloadFilename | **FALSE** |
+| relative_path | ::RDF::URI.new('http://scholarsphere.psu.edu/ns#relativePath') | **FALSE** |
+| import_url | ::RDF::URI.new('http://scholarsphere.psu.edu/ns#importUrl') | **FALSE** |
+| resource_type | ::RDF::Vocab::DC.type | TRUE |
+| creator | ::RDF::Vocab::DC11.creator | TRUE |
+| contributor | ::RDF::Vocab::DC11.contributor | TRUE |
+| description | ::RDF::Vocab::DC11.description | TRUE |
+| keyword | ::RDF::Vocab::DC11.relation | TRUE |
+| license | ::RDF::Vocab::DC.rights | TRUE |
+| rights_statement | ::RDF::Vocab::EDM.rights | TRUE |
+| publisher | ::RDF::Vocab::DC11.publisher | TRUE |
+| date_created | ::RDF::Vocab::DC.created | TRUE |
+| subject | ::RDF::Vocab::DC11.subject | TRUE |
+| language | ::RDF::Vocab::DC11.language | TRUE |
+| identifier | ::RDF::Vocab::DC.identifier | TRUE |
+| based_near | ::RDF::Vocab::FOAF.based_near | TRUE |
+| related_url | ::RDF::RDFS.seeAlso | TRUE |
+| bibliographic_citation | ::RDF::Vocab::DC.bibliographicCitation | TRUE |
+| source | ::RDF::Vocab::DC.source | TRUE |
+
+### Core metadata
+
+Core metadata properties (**_that should never be removed_**) are defined in [app/models/concerns/hyrax/core_metadata.rb](https://github.com/projecthydra-labs/hyrax/blob/master/app/models/concerns/hyrax/core_metadata.rb)
+
+| Property | Predicate | Multiple |
+| -------- | --------- | -------- |
+| depositor | ::RDF::URI.new('http://id.loc.gov/vocabulary/relators/dpt') | **FALSE** |
```

Comment:
@aaron-collier yes, but they are equivalent and readers may have an easier time if you stick to a single style here.